src/main.cpp

```
@@ -267,10 +267,14 @@ void setup() {
   }
 
   if (config.isTftEnabled()) {
-    // TFT display: runtime driver selection (st7789 or gc9a01) from NVS
+    // TFT display: runtime driver selection from NVS
     TFTDriver tftDriver = TFTDriver::ST7789;
     if (strcmp(config.getTftDriver(), "gc9a01") == 0) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)